Subject: [PATCH] Fix PackageInstaller FC when tapping on a downloaded apk notification

 DatabaseUtils: Writing exception to parcel
 DatabaseUtils: java.lang.SecurityException: Permission Denial: reading com.android.providers.downloads.DownloadProvider 
 uri content://downloads/my_downloads/831 from pid=25212, uid=10019 requires android.permission.INTERNET, or grantUriPermission()

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index f9391753..98685ac4 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -22,6 +22,8 @@
     <uses-permission android:name="android.permission.MANAGE_APP_OPS_RESTRICTIONS" />
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
     <uses-permission android:name="android.permission.HIDE_NON_SYSTEM_OVERLAY_WINDOWS" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_ALL_DOWNLOADS" />
 
     <uses-permission android:name="com.google.android.permission.INSTALL_WEARABLE_PACKAGES" />
 
